src/stores/hooks/foodFilter.ts

```diff
@@ -1,15 +1,19 @@
 import AsyncStorage from '@react-native-async-storage/async-storage'
 import { useEffect, useState } from 'react'
 
+export type FoodLanguage = 'en' | 'de' | 'default'
```

Comment:
pls extend the existing `LanguageKey` with `'default'`

Reply:
thats not possible since the existing key is not static

Reply:
```ts
type Test = LanguageKey | 'default'
```